beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -143,6 +143,7 @@ impl<AppReqId: ReqId, TSpec: EthSpec> Network<AppReqId, TSpec> {
         executor: task_executor::TaskExecutor,
         ctx: ServiceContext<'_>,
         log: &slog::Logger,
+        spec: &ChainSpec,
```

Comment:
the spec is already in the `ServiceContext` so you don't need to pass this value here